api/custom-map-ui/geoname-search-ui.html

```diff
@@ -471,11 +505,11 @@
 
             let searchPhrase = document.getElementById("query").value;
 
-            fetch('https://geogratis.gc.ca/services/geoname/en/geonames.json?q=' + searchPhrase)
+            fetch('https://geogratis.gc.ca/services/geoname/en/geonames.json?q=' + searchPhrase + "*")
                 .then(response => response.json())
                 .then(json => {
-                    document.querySelector("#results_count").innerHTML = json.items.length + " results found." + "<br>";
                     document.getElementById("results_count").style.display = "block";
+                    document.querySelector("#results_count").innerHTML = json.items.length + " results found." + "<br>";
```

Comment:
I just want to note that `<br>` is a semantic line break, see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns for more info.